ui/src/App.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import {injectIntl} from 'react-intl';
+import {BrowserRouter, Route} from "react-router-dom";
+
+import Create from "./pages/Create.js";
+import GlobalHome from "./pages/GlobalHome.js";
+import RepoHome from "./pages/RepoHome.js";
+import Results from "./pages/Results.js";
+import View from "./pages/View.js";
+
+import './css/all.scss';
+
+const App = () => (
+  <BrowserRouter>
+    <div>
+      /**
```

Comment:
Use a normal comment // ... instead of JSDoc /** ... */ for this.

Reply:
Done.